### PR TITLE
Add preview relation type #1090

### PR DESCRIPTION
**Pull request by m-mohr**
**Related Issue(s):** #1090, https://github.com/opengeospatial/ogcapi-records/issues/94

**Proposed Changes:**

1. Add a new relation type to best practices for a preview, see PR for more details.

**PR Checklist:**

- [x] This PR is made against the dev branch (all proposed changes except releases should be against dev, not master).
- [x] This PR has **no** breaking changes.
- [x] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-spec/blob/dev/CHANGELOG.md) **or** a CHANGELOG entry is not required.
- [ ] This PR affects the [STAC API spec](https://github.com/radiantearth/stac-api-spec), and I have opened issue/PR #XXX to track the change.

### CODE REVIEW

**Review by cholmes — COMMENTED** [review]
Looks like you just copied the line from 'next' but didn't change the text in the description?

**Comment by m-mohr** [comment]
This is ready for review now.

**Review by cholmes — APPROVED** [review]

File: best-practices.md [hunk 1]
```diff
@@ -627,13 +628,14 @@ with the `type` field) to communicate the structure and content of related entit
...
 | via       | The URL of the source metadata that this STAC Item or Collection is created from. Used similarly to canonical, but refers back to a non-STAC record (Landsat MTL, Sentinel tileInfo.json, etc) |
 | prev      | Indicates that the link's context is a part of a series, and that the previous in the series is the link target. Typically used in STAC by API's, to return smaller groups of Items or Catalogs. |
 | next      | Indicates that the link's context is a part of a series, and that the next in the series is the link target. Typically used in STAC by API's, to return smaller groups of Items or Catalogs. |
+| preview   | Refers to a resource that serves as a preview (see [RFC 6903, sec. 3](https://tools.ietf.org/html/rfc6903#section-3)), usually a lower resolution thumbnail. In STAC this would usually be the same URL as the [thumbnail](#thumbnail) asset, but adding it as a link in addition enables OGC API clients that can't read assets to make use of it. It also adds support for thumbnails to STAC Catalogs as they can't list assets. |
```
Comment (jisantuc) [inline best-practices.md:638]: Is the bit about catalog previews relevant? I understand that it's technically true, but it feels sort of orthogonal to the explanation of previews

**Review by jisantuc — APPROVED** [review]

File: best-practices.md (hunk 1 above)
Comment (m-mohr) [reply best-practices.md:638]: I felt like making clear that it is relevant as this is (in contrast to Items and Collections) the only way to add a preview for Catalogs. But maybe it isn't.

Comment (cholmes) [reply best-practices.md:638]: If this was in the spec itself I think it'd make sense to not have so much here, but I think it's a decent point to make in the best practice document.

**Pull request merged**